app/models/metering.rb

```diff
@@ -40,7 +57,7 @@ def calculate_costs(consumption, _)
       end
 
       chargable_field = ChargeableField.find_by(:group => group, :source => source)
-      next if field == "existence_hours_metric" || field == "fixed_compute_metric" || chargable_field && chargable_field.metering?
+      next if field == 'metering_allocated_cpu_cores_metric' || field == 'metering_allocated_cpu_metric' || field == 'metering_allocated_memory_metric' || field == "existence_hours_metric" || field == "fixed_compute_metric" || chargable_field&.metering?
```

Comment:
Maybe you can simplify this by putting the list of a"allocated" fields in an array an doing a look up of `field` in ti?

Reply:
yes it is better, I added it 👍